lib/node-owners.js

```diff
@@ -1,9 +1,6 @@
 'use static'
 
-const { parse } = require('codeowners-utils')
-// NOTE(mmarchini): `codeowners-utils` doesn't respect ./ prefix,
-// so we use micromatch
-const micromatch = require('micromatch')
```

Comment:
This can be uninstalled now, right?

Reply:
Right, forgot to uninstall before committing. Will do it before landing